lms/static/sass/utilities/_extends.scss

```diff
@@ -0,0 +1,293 @@
+// LMS -- utilities -- extends
```

Comment:
TODO: I'll talk to someone who's a Git ninja about why this file was deleted/re-created vs. moved in the Git history.